src/k8s/pkg/client/kubernetes/pods_test.go

```diff
@@ -0,0 +1,107 @@
+package kubernetes
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/kubernetes/fake"
+	k8stesting "k8s.io/client-go/testing"
+)
+
+func TestCheckForReadyPods(t *testing.T) {
+	g := NewGomegaWithT(t)
```

Comment:
`NewGomegaWithT` is deprecated for `NewWithT`
```suggestion
	g := NewWithT(t)
```

Also, each test needs its own gomega instance (otherwise test failures will panic)